VulkanLaunchpad.h

```diff
@@ -330,6 +330,19 @@ vklAllocateHostCoherentMemoryForGivenRequirements(VkDeviceSize bufferSize, VkMem
  */
 VkBuffer vklCreateHostCoherentBufferWithBackingMemory(VkDeviceSize buffer_size, VkBufferUsageFlags buffer_usage);
 
+/*!
+ *	Creates a new buffer (VkBuffer) and also allocates new memory on the device (VkDeviceMemory) to back the
+ *	buffer's size requirements. The memory will always be allocated from a region of so called "device-local"
+ *	memory, which is a region that is not accessible from the CPU, so that GPU can work faster on it.
+ *	Internally, this is indicated with the flag VK_MEMORY_PROPERTY_DEVICE_LOCAL_BIT.
```

Comment:
After combination of the two functions into one, this should be:
```cpp
 *	Creates a new buffer (VkBuffer) and also allocates new memory to back the
 *	buffer's size requirements. the memory is managed internally by Vulkan Launchpad. 
 * 
 *	@note    To allocate a buffer in host-coherent memory (which is a region that both, the
 *	              CPU and the GPU have access to, which can be useful for tranferring data every
 *	              frame to GPU, or general for data transfer from/to GPU, pass two flags to the flags 
 *	              parameter: VK_MEMORY_PROPERTY_HOST_VISIBLE_BIT | VK_MEMORY_PROPERTY_HOST_COHERENT_BIT
 * 
 *	@note    To allocate a buffer in device-local memory (which is a region that is not 
 *	              accessible from the CPU, so that GPU can work faster on it.), pass the flag
 *	               VK_MEMORY_PROPERTY_DEVICE_LOCAL_BIT to the flags parameter!
```